docs/source/roadmap.rst

```diff
@@ -1,9 +1,52 @@
 =======
 Roadmap
 =======
-This page describes the high-level roadmap for the MOTECH 1.0 release. For more granular and up-to-date information about release plans, click on the issue tracker links for each of the releases below.
+This page describes the high-level roadmap for the MOTECH system as we move toward the release of version 1.0. For more granular and up-to-date information about release plans, click on the issue tracker links for each of the releases below.
 
-Version 1.0 - Release date: Q1 2016
+Version 0.28 - Release date: April 2016
+=====================================
+
+`Issue Tracker <https://applab.atlassian.net/browse/MOTECH/fixforversion/20840/?selectedTab=com.atlassian.jira.jira-projects-plugin:version-issues-panel>`_
+
+Version 0.28 will represent a breaking change between 0.27. We plan to upgrade a number of core components that will require changes in deployment.
+
+Key Technical Changes
+---------------------
+The following is a list of key technical changes planned to be implemented in version 0.28:
+    - Upgrade to Java 8
+    - Upgrade Data Nucleus to version 4.0.1
+    - Improve UI build process with Node, Bower, Gulp and SCSS
+    - Split database from motech_data_services to motech_data and motech_schema
+
+
+Module Enhancements
+-------------------
+    - New IHE Interop Module supports the creation of HL7 CDA documents in MOTECH
+    - Improved OpenMRS module with tasks integration
+    - New Atom-Client module allows MOTECH to query atom feeds and act on changes in the feed. It will primarily be used in OpenMRS integration to detect when a new patient, encounter or provider has been created. However, this module can work with any atom feed.
+
```

Comment:
I verified this afternoon that the atom client module is working as intended and commented on the ticket my findings with a cc to you and Frank.